src/api/Authentication.js

```diff
@@ -0,0 +1,24 @@
+export class Authentication{
+  constructor(){}
+
+  static validityChecker(email, password){
+    const validPassword = Authentication.passwordChecker(password);
+    const validEmail = Authentication.emailChecker(email);
+    if(validEmail && validPassword) {
+      localforage.setItem('authenication', 'Successful');
+    }
+    return 'Successful';
+  }
+
+  static passwordChecker(password){
+    const trimedPassword = password.trim();
+    if(!trimedPassword.length) return 'Password cannot be empty';
+    else if(trimedPassword.length === 1 || password.length ===2) return 'Invalid password, length of 3 at least';
```

Comment:
***Code Quality: Nitpick***
This looks good but it would be neater if there is no else statement at all
so this could still work like this
```if(trimedPassword.length === 1 || password.length ===2) return 'Invalid password, length of 3 at least'```